src/embeddings/embeddings_api.cpp

```diff
@@ -37,12 +38,26 @@ std::variant<EmbeddingsRequest, std::string> EmbeddingsRequest::fromJson(rapidjs
         if (it->value.IsString()) {
             input_strings.push_back(it->value.GetString());
         } else if (it->value.IsArray()) {
+            int index = 0;
             for (auto& input : it->value.GetArray()) {
-                // TODO: is array of ints
-                // TODO: is int
-                if (!input.IsString())
-                    return "every element in input array should be string";
-                input_strings.push_back(input.GetString());
+                if (input.IsArray()) {
+                    std::vector<int> ints;
+                    for (auto& val : input.GetArray()) {
+                        if (val.IsInt())
+                            ints.push_back(val.GetInt());
```

Comment:
Looks like this deserialization allows:
```
[ [62, 71, True, 5, "abc", 1 ], ... ]
```
is it desired?

Reply:
@bstrzele

Reply:
this will not be successfully deserialized https://github.com/openvinotoolkit/model_server/pull/2775/files#diff-2b9c77dffac8a9598e047bebd0ccac92c2e74e198d0664a90f9944d536f6b9beR195